sjs-bundler/src/main/scala/lucuma/sbtplugin/LucumaSJSBundlerPlugin.scala

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.sbtplugin
+
+import sbt._, Keys._
+import scalajsbundler.sbtplugin.ScalaJSBundlerPlugin
+import _root_.io.circe.Decoder
+import _root_.io.circe.jawn
+
+object LucumaSJSBundlerPlugin extends AutoPlugin {
+
+  override def requires = ScalaJSBundlerPlugin
+
+  override def trigger = allRequirements
+
+  import ScalaJSBundlerPlugin.autoImport._
+
+  override lazy val projectSettings = Seq(
+    Compile / npmDependencies ++= {
+      val log = sLog.value
+      readPackageJson(baseDirectory.value, log)(_.dependencies) ++
+        readPackageJson((LocalRootProject / baseDirectory).value, log)(_.dependencies)
```

Comment:
I suppose these should be reversed, so that the `package.json` in the module dir takes precedence.